test/dlc_tests/conftest.py

```diff
@@ -996,6 +1001,7 @@ def lookup_condition(lookup, image):
     """
     # Extract ecr repo name from the image and check if it exactly matches the lookup (fixture name)
     repo_name = get_ecr_repo_name(image)
+    LOGGER.info(f"[logs-ABCD][Displaying] repo_name: {repo_name} lookup: {lookup} image: {image}")
```

Comment:
Can you please remove these lines in this file that start with `LOGGER.info(f"[logs-ABCD].....` in the next commit? 

I added these lines for logging purposes and did not remove till the fix was in place.

Since the issue is now fixed, can you please remove it?

Reply:
Done